nginx-oss-apb/roles/provision-nginx-oss-apb/tasks/main.yml

```diff
@@ -0,0 +1,98 @@
+##############################################################################
+## Provision NGINX OSS APB
+## This role executes much of the needed functionality to provision an
+## application using an Ansible Playbook Bundle.  Included in the comments
+## below are some sample resources for getting started deploying an application
+## to OpenShift.
+##############################################################################
+
+##############################################################################
+## An OpenShift Origin deployment configuration provides a replication
+## controller, spins up pods, and also provides the ability to transition from
+## one deployment of an image to a new one.
+## https://docs.openshift.org/latest/architecture/core_concepts/deployments.html#deployments-and-deployment-configurations
+##############################################################################
+- name: Create NGINX OSS APB deployment config
+  openshift_v1_deployment_config:
+    name: nginx-oss-apb
+    namespace: '{{ namespace }}'
+    state: present
+    labels:
+      app: '{{ namespace }}'
+      service: nginx-oss-apb
+    replicas: 1
+    selector:
+      app: '{{ namespace }}'
+      service: nginx-oss-apb
+    spec_template_metadata_labels:
+      app: '{{ namespace }}'
+      service: nginx-oss-apb
+    containers:
+      - image: docker.io/alessfg/openshift-nginx
+        name: nginx-oss-apb
+        securityContext:
+          runAsUser: 1000090000
```

Comment:
I have tested this PR while removing lines 33-34 and it worked great on my cluster. I am not sure if you need it to run in your cluster or not but I would prefer not specifying an scc uid unless we know it can work in a generic cluster that is running the restricted scc. Please let me know if your tests succeed while removing these lines.

Reply:
Thanks for testing it @dymurray - will test on my end before EOD